### PR TITLE
Make @pixi/unsafe-eval self-installed

### PR DESCRIPTION
**Pull request by SuperSodaSea**
Since one rarely import `@pixi/unsafe-eval` without installing it, I'd like to make `@pixi/unsafe-eval` self-installed.

##### Description of change

- `@pixi/unsafe-eval` is self-installed when imported, so that `install()` no longer needs to be called manually
    - Deprecate `install()` in `@pixi/unsafe-eval`
- `unsafeEvalSupported()` is exported from `@pixi/core`
    - `@pixi/unsafe-eval` is only installed if `unsafeEvalSupported()` returns `false`

##### Pre-Merge Checklist

- [x] Documentation is changed or added
- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

**Review by bigtimebuddy — APPROVED** [review]
I was thinking this same thing when I was looking at this recently! Thanks for this!

**Comment by bigtimebuddy** [comment]
Out of curiosity, do you use this package in any projects or on any platforms?

**Comment by SuperSodaSea** [comment]
> Out of curiosity, do you use this package in any projects or on any platforms?

In my personal project, which is still in a very early stage :)

**Comment by SuperSodaSea** [comment]
Another example of using `@pixi/unsafe-eval` is when using PixiJS in Electron. Electron will show a CSP warning if `unsafe-eval` is enabled (See https://content-security-policy.com/examples/electron/). I don't like any warning in the console, so I need to disable `unsafe-eval` and use `@pixi/unsafe-eval`.

File: packages/unsafe-eval/src/install.ts [hunk 1]
```diff
@@ -1,39 +1,53 @@
...
+ * @param _core
+ * @deprecated since 7.1.0
+ */
+export function install(_core: PIXICore): void
 {
-    if (!ShaderSystem)
-    {
-        throw new Error('Unable to patch ShaderSystem, class not found.');
-    }
+    utils.deprecation('7.1.0', 'install() has been deprecated, @pixi/unsafe-eval is self-installed since 7.1.0');
```
Comment (bigtimebuddy) [inline packages/unsafe-eval/src/install.ts:19]: One small thing , can you wrap this in `// #if _DEBUG` comment? This makes sure it's excluded from the mini field build.

Comment (SuperSodaSea) [reply packages/unsafe-eval/src/install.ts:19]: Sure, I will fix this later.

Comment (SuperSodaSea) [reply packages/unsafe-eval/src/install.ts:20]: Fixed!

**Pull request merged**